cx_Freeze/command/bdist_msi.py

```diff
@@ -194,6 +195,7 @@ def add_config(self) -> None:
                 ("A_SET_TARGET_DIR", 'TARGETDIR=""', 401),
                 ("A_SET_REINSTALL_MODE", 'REINSTALLMODE=""', 402),
                 ("SelectDirectoryDlg", "not Installed", 1230),
+                ("LicenseAgreementDlg", "not Installed", 1240),
```

Comment:
Note: this doesn't require the dlg be included to be valid

Reply:
Do you mean that, if there is no license file, the dialog will not appear?

Reply:
Exactly